### PR TITLE
l10n: Change to lowercase

### PR DESCRIPTION
**Pull request by Valdnet**
Unifying the text string in the Nextcloud instance.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: templates/stream.app.navigation.php [hunk 1]
```diff
@@ -55,7 +55,7 @@
 			</span>
 			<div id="activity-personal-settings-link">
 				<a href="<?php p($_['personalSettingsLink']); ?>">
-					<span class="no-icon"><?php p($l->t('Personal Activity Settings')); ?></span>
+					<span class="no-icon"><?php p($l->t('Personal activity settings')); ?></span>
```
Comment (szaimen) [inline templates/stream.app.navigation.php:58]: Maybe we should rename it to Personal notification settings on NC23 and above?

Comment (Valdnet) [reply templates/stream.app.navigation.php]: Very good idea 👍. More suited to the type of setup. I changed 😉.

**Review by szaimen — APPROVED** [review]

**Review by artonge — APPROVED** [review]

**Comment by szaimen** [comment]
Manual backport in https://github.com/nextcloud/activity/pull/761

**Pull request merged**